web/src/server_events_dispatch.js

```diff
@@ -310,6 +305,11 @@ export function dispatch_normal_event(event) {
                                     realm["realm_" + key] = value;
                                 }
 
+                                if (key === "topics_policy") {
+                                    compose_recipient.update_topic_inputbox_on_topics_policy_change();
+                                    compose_recipient.update_compose_area_placeholder_text();
+                                }
```

Comment:
I think this can use the `realm_settings[key]()` framework of calling registered helpers, rather than each feature adding a call here. I guess this should be documented better, since we've got a dozen violations of that model, but the intent is that one registers a function above rather than passing `noop`. Can be a follow-up PR to clean this up more fully, so let's not change this for this PR.